api/src/pcapi/core/offers/exceptions.py

```diff
@@ -173,6 +173,11 @@ def __init__(self, error_message: str) -> None:
         super().__init__("ean", error_message)
 
 
+class LabelTooLong(OfferCreationBaseException):
```

Comment:
(détail) "label" est peut-être un peu trop vague.

```suggestion
class PriceCategoryLabelTooLong(OfferCreationBaseException):
```